README.md

```diff
@@ -1,322 +1,329 @@
-
-This driver can only be used with FQL X, and is not compatible with earlier versions of FQL. This README contains detailed instructions on how to use the driver, split into the following sections .
-
-# Quick Start for the FQL X driver
-
-## Set-up and Dependencies
+See the [Fauna Documentation](https://fqlx-beta--fauna-docs.netlify.app/fqlx/beta/) for additional information how to configure and query your databases.
 
-This driver is on NPM.
+This driver can only be used with FQL X, and is not compatible with earlier versions of FQL. To query your databases with earlier API versions, see the [faunadb](https://www.npmjs.com/package/faunadb) package.
 
-You can install with npm with `npm install fauna`.
-You can install with yarn with `yarn add fauna`
+<details>
+<summary>Table of Contents</summary>
 
-## Pre-requisites
+- [A JavaScript driver for Fauna.](#a-javascript-driver-for-fauna)
+- [Compatibility](#compatibility)
+- [Installation](#installation)
+  - [Node.js](#nodejs)
+  - [Browsers](#browsers)
+- [Usage](#usage)
+  - [Connecting from the browser](#connecting-from-the-browser)
+  - [Importing into a bundled project](#importing-into-a-bundled-project)
+  - [Quick-Start](#quick-start)
+  - [Querying with FQL X](#querying-with-fql-x)
+  - [Query Composition](#query-composition)
+  - [Typescript Support](#typescript-support)
+  - [Query Options](#query-options)
+  - [Client Configuration](#client-configuration)
+    - [Using environment variables](#using-environment-variables)
+  - [Query Statistics](#query-statistics)
+- [Contributing](#contributing)
+  - [Setting up this Repo](#setting-up-this-repo)
+  - [Running tests](#running-tests)
+  - [Linting your code](#linting-your-code)
+- [License](#license)
 
-If you have not already created a database and access key (secret) in the Preview environment please do so now. Contact product@fauna.com for access to Preview.
+</details>
 
-## Creating a connection to your database
+# Compatibility
 
-The following code example sets up a connection to Fauna's Preview environment, authenticates with the secret you provided, and executes a simple query.
+This Driver supports and is tested on:
 
-```typescript
-import { Client, endpoints, fql } from "fauna";
-
-const client = new Client({
-  endpoint: endpoints.preview,
-  secret: "<my_fauna_secret>",
-});
-```
+- Node.js [Current and Active LTS](https://nodejs.org/en/about/releases/)
+  - Current - v19
+  - Active LTS - v18
+- Chrome
+- Firefox
+- Safari
+- Edge
 
-Default settings are available for the client. So you can creat a client as simply as:
+# Installation
 
-```typescript
-import { Client } from "fauna";
+## Node.js
```

Comment:
Node.js feels misleading.

You don't have to be executing on node to install with npm or yarn.

I'd call this section ## Installing

```suggestion
## Installing
The fauna-js driver is available on npm.

You can install with your package manager of choice:

* `npm install fauna`
* `yarn add fauna`
```